### DISCUSSION TITLE
Migration from dev to prod, possible?

### DISCUSSION THREAD

**Discussion by Bbillyben** [inventree/InvenTree#3734] [Q&A] [answered]
Hi there, 

I'm trying to move from a typical docker dev environnement to a production one (still on 'latest' tag by the way).

I've done the export from my dev env : 

`sudo docker-compose run inventree-dev-server invoke export-records -f expdata.json  `

I've set up my prod env (which works well as far as I can tell), in a brand new folder dedicated to prod, move the json 

and trying to import datas : 

`sudo docker-compose run inventree-server invoke import-records -c -f expdata.json`

but wherever I can cp the json and setting the right path, i land on 

```
[+] Running 1/0
⠿ Container inventree-db  Running                                                                                                               0.0s
/home/inventree/data/config.yaml exists - skipping
InvenTree translation coverage: 19%
Error: File /home/inventree/expdata.json' does not exist
```

I've tryed to import in the dev environnement, and it works  easily.
I've updated (invoke udate) on both and re-export/import, but this error still  pop.

Is there anything I missed?

[Edit] I can link my docker-compose if needed or my .env.

DB user and passwords are differents from dev to prod.

**Comment by SchrodingersGat** [comment, marked as answer]
You have to specify the path to the data file relative to the container environment. If you have placed the file in the external "data" volume, that command *should* be:

`sudo docker-compose run inventree-server invoke import-records -c -f data/expdata.json`

**Reply by Bbillyben** [reply]
thanks @SchrodingersGat ,


And sorry I should have had a more complete description. For the records here is what I've tested before :

I've tryed to specify :
 * an absolute path : `/stock/ivlprod/expdata.json`)
 * a path relative to the current folder : ./expdata.json
 * a user path : ~/expdata (moving the json of course)
 * and maybe some other

and i even test to copy in the data sub folder (set as the docker volume), but my command  history let me know i've miserably fail to point to the file........(many times......)


**But** I've just re-cp in data/ and invoke import with the right relative path, and it went to the whole process. 
note 
```
cp expdata.json data/
sudo docker-compose run inventree-server invoke import-records -c -f data/expdata.json
```


if I still had hair, I would rip them out


The unexplained remain to me is why the import from the current local directory has work in dev and not in prod?!
